arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/signals/Exceptions.java

```diff
@@ -57,7 +57,7 @@ public static boolean noStackException(final short bitmask) {
   }
 
   /**
-   * @return true if any exception flag has been raised
+   * @return true if any exception flag has been raised, i.e., at least one exception flag is raised
```

Comment:
We should also look into the stack exceptions that are caught by the HUB before it ever gets to instruction processing. AFAICT these aren't covered in the above. These coverage issues really push me in the direction of implementing the `UNDEFINED` value in the `TracedException` class.

Reply:
I made the same comment before 😇

Reply:
@OlivierBBB is this solved by this in `CommonFragmentValues` or it is something we should check even earlier?

```
   tracedException = noException ? TracedException.NONE : TracedException.UNDEFINED;

    if (instructionFamily == INVALID) {
      checkArgument(Exceptions.invalidOpcode(exceptions));
      setTracedException(TracedException.INVALID_OPCODE);
    }

    if (Exceptions.stackUnderflow(exceptions)) {
      setTracedException(TracedException.STACK_UNDERFLOW);
    }

    if (Exceptions.stackOverflow(exceptions)) {
      setTracedException(TracedException.STACK_OVERFLOW);
    }
```

Reply:
I imagine the above will have solved this. We have to trace a `CommonFragment` in all cases, whether we have a stack exception or not. So this should be sufficient.

Reply:
In case of an exception, we create a StackOnlySection, so imho, it's better to set for INVALID / SUX / SOX this tracedException before the OOGX. 
![image](https://github.com/user-attachments/assets/20d2f8bd-fda4-4b79-aee6-3d34a444761c)

Reply:
@OlivierBBB @letypequividelespoubelles what should we agree on?

Reply:
I am not understanding your last comment. Could you please rephrase it @letypequividelespoubelles ?

Reply:
My understanding of our previous discussion was that doing it in the common section may not be the best since we would be doing the associated computations in execution phases where it doesn't matter (TX_INIT, TX_FINL, TX_SKIP). Was this what you meant @letypequividelespoubelles or did you mean something else ?

Reply:
> We should also look into the stack exceptions that are caught by the HUB before it ever gets to instruction processing.

In case of exceptions caught by the HUB before processing the opcode (INVALID, SUX and then SOX), we will create a `StackOnlySection` . IMHO, it would be better to set the tracedException in the `StackOnlySection` instead of setting it in the `CommonFragment`.
Impact : 
- more consistent with other BlaBlaSection : this is where we set the tracedException in all case
- tiny tiny better perf : we'll do the 3 checks check only for StackSection, and not for all section (so divide by ~ 2 the number of checks, as we have roughly half sectiuons which are StackOnlySection

Reply:
Note that I'm talking about StackOnlySection, not StackFragment

Reply:
And if we really want to reduce the number of check, we could introduce a `PrecoceExceptionSection`. Let me push what I have in mind.